mewarpx/mewarpx/setups_store/diode_setup.py

```diff
@@ -546,14 +546,48 @@ def init_traceparticles(self):
         print('### Init Diode TraceParticles ###')
 
     def init_runinfo(self):
-        raise NotImplementedError(
-            "Diode Runinfo is not yet implemented in mewarpx")
         print('### Init Diode Runinfo Setup ###')
 
+        runvars = DiodeRun_V1.__dict__.copy()
+        runvars.update(self.__dict__)
+
+        injector_dict = {'cathode': self.injector}
+        surface_dict = {
+            'cathode': self.cathode,
+            'anode': self.anode,
+            # TODO: implement accgrid, absgrid, dielectric
+            # 'accgrid': self.acc_grid,
+            # 'absgrid': self.absorption_grid,
+            # 'dielectric': [self.SiO2_1, self.SiO2_2],
+        }
+        # Output runinfo
+        self.runinfo = runinfo.RunInfo(
+            injector_dict=injector_dict,
+            surface_dict=surface_dict,
+            local_vars=runvars,
+            run_file=__file__,
+            # TODO: implement collecting run parameters
+            # run_param_dict=self.setupinfo.run_param_dict,
+            electrode_params={
+                "CATHODE_A": self.CATHODE_A,
+                "CATHODE_TEMP": self.CATHODE_TEMP,
+                "CATHODE_PHI": self.CATHODE_PHI,
+                "ANODE_PHI": self.ANODE_PHI
+            },
+        )
+        self.runinfo.save()
+
     def init_fluxdiag(self):
-        raise NotImplementedError(
-            "Diode FluxDiag is not yet implemented in mewarpx")
         print('### Init Diode FluxDiag ###')
+        self.fluxdiag = flux_diagnostic.FluxDiagnostic(
+            diag_steps=self.DIAG_STEPS,
+            write_dir="diags/fluxes",
+            # TODO: pass scraper when scraper is added
+            scraper=None,
```

Comment:
These lines can be removed as well since we won't actually pass a scraper object to `FluxDiagnostic` anymore.